projects/EfficientDet/REMEAD.md

```diff
@@ -0,0 +1,128 @@
+<div align="center">
+
+# EfficientDet: Scalable and Efficient Object Detection
+
+<br>
+Mingxing Tan, Ruoming Pang, Quoc V. Le
+<br>
+<a href="https://arxiv.org/pdf/1911.09070.pdf">[arXiv paper]</a>
+<br>
+</div>
+
+## Description
+
+This is an implementation of [EfficientDet](https://github.com/google/automl) based on [MMDetection](https://github.com/open-mmlab/mmdetection/tree/3.x), [MMCV](https://github.com/open-mmlab/mmcv), and [MMEngine](https://github.com/open-mmlab/mmengine).
+<br>
+EfficientDet a new family of object detectors, which consistently achieve much better efficiency than prior art across a wide
+spectrum of resource constraints.
+<br>
+In particular, with singlemodel and single-scale, EfficientDet-D7 achieves stateof-the-art 55.1 AP on COCO test-dev with 77M parameters and 410B FLOP
+<br>
+BiFPN is a simple yet highly effective weighted bi-directional feature pyramid network, which introduces learnable weights to learn the importance of different input features, while repeatedly applying topdown and bottom-up multi-scale feature fusion.
+<br>
+In contrast to other feature pyramid network, such as FPN, FPN + PAN, NAS-FPN, BiFPN achieves  the best accuracy with fewer parameters and FLOPs.
+
+<div align="center">
+<img src="https://github.com/zwhus/pictures/raw/main/Screenshot%20from%202023-01-30%2015-33-42.png">
+</div>
+<br>
+
+## Usage
+
+### convert weights from tensorflow to pytorch
+
+```bash
+python efficientdet/transfer_efficientdet --backbone {backname} --tensorflow_weight {tensorflow_efficientdet_path} --out_weight {out_path}
+```
+
+### For Example
+
+```bash
+python efficientdet/transfer_efficientdet --backbone efficientnet-b0 --tensorflow_weight efficientdet-d0/model --out_weight demo.pth
```

Comment:
- we can move this file to `projects/EfficientDet/transfer_efficientdet.py`
- better to rename it as convert_tf_to_pt.py
- suggest to indicate users they need to install tensorflow if they want to use this script.